src/gui/MatchTemplatePanel.h

```diff
@@ -40,6 +40,22 @@ class MatchTemplatePanel : public MatchTemplatePanelParent {
     bool was_enabled_DefocusSearchRangeNumericCtrl;
     bool was_enabled_DefocusSearchStepNumericCtrl;
 
+    // Save the currently set values of the controls.
+    // These are *not* set until a call to SetInputsForPossibleReRun(true)
+    bool     was_values_set = false;
```

Comment:
suggest: swap was_ for previous_

Reply:
I actually ended up not using these. We might discuss with the lab whether they want these restored... My feeling is that it's not very useful. We might think about having the possibility to store custom default values in the database.